### PR TITLE
ISSUE #868 - Bind issue functions to the issues component scope

### PR DESCRIPTION
**Pull request by JamesMilnerUK**
Use bind to preserve scope. This is probably the cause of any current issue / chat server errors.

### CODE REVIEW

File: frontend/components/issues/js/issues.component.ts [hunk 1]
```diff
@@ -233,22 +232,21 @@ class IssuesController implements ng.IController {
 
 				this.issuesReady.then(() => {
 					const hasPerm = this.AuthService.hasPermission(
-						this.ClientConfigService.permissions.PERM_CREATE_ISSUE, 
-						this.modelSettings.permissions
+						this.ClientConfigService.permissions.PERM_CREATE_ISSUE,
```
Comment (carmenfan) [inline frontend/components/issues/js/issues.component.ts]:
````js
this.canAddIssue = this.AuthService.hasPermission(....)
````

Does this have latest of staging? I feel like we've changed this already on a separate branch

Comment (JamesMilnerUK) [reply frontend/components/issues/js/issues.component.ts]: That was for groups :P The reason this code is even highlighted is auto formatting by TSLint

Comment (carmenfan) [reply frontend/components/issues/js/issues.component.ts]: no I mean i thought we've got rid of the hasPerm variable and just assigned this.canAddIssue directly somewhere already. I remember pointing this out elsewhere

Comment (JamesMilnerUK) [reply frontend/components/issues/js/issues.component.ts]: Yes it was very similar looking code but different file;  ```this.canAddGroup```  vs ```this.canAddIssue```. Probably worth changing it here to, I can look into it.

**Pull request merged**